ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -100,17 +100,16 @@ func listContainers(c *cli.Context) ([]types.Container, error) {
 	return listLocalComposeContainers(localproject.LocalOutDefaultFileName)
 }
 
-func listLocalComposeContainers(composePath string) ([]types.Container, error) {
-	wd, _ := os.Getwd()
-	if _, err := os.Stat(filepath.Join(wd, localproject.LocalOutDefaultFileName)); os.IsNotExist(err) {
+func listLocalComposeContainers(basePath string) ([]types.Container, error) {
+	if _, err := os.Stat(basePath); os.IsNotExist(err) {
```

Comment:
I think this cannot check the error for the case if the file cannot be open because of some other restriction (see [link](https://stackoverflow.com/questions/12518876/how-to-check-if-a-file-exists-in-go)). Do you think it would be better to be:

```
if _, err := os.Stat(basePath); err != nil {
    logrus.Fatalf("fail to open compose file %s due to\n%v", localproject.LocalOutDefaultFileName, err)
}
```

Reply:
👍 yes! Done